### PR TITLE
Jk documentation

### PR DESCRIPTION
**Pull request by kearneyjj1**
Add all documentation changes.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -86,6 +86,15 @@
         "doc"
       ]
     },
+    {
+      "login": "sudhirtibrewal",
+      "name": "Sudhir Tibrewal",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/12657704?v=4",
+      "profile": "https://github.com/sudhirtibrewal",
+      "contributions": [
+        "code"
```
Comment (sudhirtibrewal) [inline .all-contributorsrc]: you can also add  "ideas"m, which is planning

File: README.md [hunk 2]
```diff
@@ -3,22 +3,30 @@
...
-Join us on our [Discord](https://discord.gg/anTP7xm) for any questions and discussions.
+
+
+**Important Release Notes 31/03/2020**
+
+***Known UDP messaging issue***
+For the initial release of the Catalyst Network UDP is used as the peer to peer messaging protocol. This causes fragmentation of packages and thereby transactions over 1280Bytes are prone to packet loss as discussed in [Issue #909](https://github.com/catalyst-network/Catalyst/issues/909). Due to the DFS utilising a seperate TCP messaging system this does not affect files stored in the DFS including ledger state updates. However it will affect both simple and smart contract transactions. 
+
+***MacOS node running issue***
+Currently there is an issue with running RocksDB on MacOS meaning that a node can not unfortunatly be run on MacOS. A fix is currently in progress and will be availiable iminently.
```
Comment (sudhirtibrewal) [inline README.md]: eminently

File: .all-contributorsrc (hunk 1 above)
Comment (kearneyjj1) [reply .all-contributorsrc]: No worries, Ill make this change.

File: README.md (hunk 2 above)
Comment (kearneyjj1) [reply README.md]: Sorry referring to what part?

**Pull request merged**